crates/iota/src/validator_commands.rs

```diff
@@ -503,7 +505,7 @@ async fn get_cap_object_ref(
         let owner = resp.owner().unwrap();
         let cap_obj_ref = resp
             .object_ref_if_exists()
-            .unwrap_or_else(|| panic!("OperationCap {cap_object_id} shall exist."));
+            .unwrap_or_else(|| panic!("OperationCap {cap_object_id} does not exist"));
```

Comment:
Just passing by, don't mind me :P
```suggestion
            .with_context(|| format!("OperationCap {cap_object_id} does not exist"))?;
```

Reply:
The only reason I don't like this is that it won't print the stack trace, and in the case that this happens something is very wrong